site/sk.njk

```diff
@@ -0,0 +1,161 @@
+                občania tejto zeme podnikli kroky a urobili svoj kúsok na podielaní sa v boji proti COVID-19.
+                <br /><br />
+                Povedzme to otvorene: <strong>Zostaňte do riti doma!</strong>
+            </p>
+
+            <h2 class="text-3xl">Samo-karanténne Prehlásenie</h2>
+
+            <p>
+                Bez dobre preštudovanej liečby a nedostupnej vakcíny najskôr do budúceho roka, jediný učinný spôsob
+                ako udržať pandémiu koronavírus v prístave, je dať vírusu, čo najmenšie šance na jeho šírenie sa.
+                Nasledujúci zoznam akcií, zoradených od tých, ktoré sa dajú najjednoduchšie implementovať, až po tie
+                najefektívnejšie v boji proti pandémií, by mal slúžiť ako súbor usmernení pre ľudí, ktorí sa chcú pripojiť
+                a spraviť potrebné kroky, ktoré môžu skutočne zmeniť stav.
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Neprepadajte panike, ale buďte opatrní.</li>
+                <li class="pb-2">Umývajte si ruky často a praktikujte dobrú etiketu proti kašľu a kýchaniu.</li>
+                <li class="pb-2">Pokúste sa, čo najmenej dotýkať tváre, vrátane úst, nosa a očí.</li>
+                <li class="pb-2">Precvičujte sociálne dištancovanie, žiadne objatia a bozky, žiadne podanie ruky,
+                žiadne "high fives". Ak je to potrebné, použite bezpečnejšie alternatívy.</li>
+                <li class="pb-2">Nezúčastňujte sa koncertov, divadelných hier, športových udalostí ani iných hromadných
+                zábavných podujatí.</li>
+                <li class="pb-2">Vyhnite sa návšteve múzeí, výstav, kín, nočných klubov a ďalších zábavných zariadení.</li>
+                <li class="pb-2">Držte sa ďalej od spoločenských stretnutí a podujatí, ako sú klubové stretnutia,
+                náboženské služby a súkromné večierky.</li>
+                <li class="pb-2">Znížte cestovanie na minimum. Ak to nie je absolútne nevyhnutné, nejazdite na veľké
+                vzdialenosti.</li>
+                <li class="pb-2">Ak to nie je absolútne nevyhnutné, nepoužívajte verejnú dopravu.</li>
+                <li class="pb-2">Ak môžete pracovať z domu, pracujte z domu. Požiadajte svojho zamestnávateľa, aby vám v
+                prípade potreby umožnil prácu na diaľku.</li>
+                <li class="pb-2">Nahraďte, čo najviac socíálnych interakcií vzdialenými alternatívami, ako sú telefónne
+                hovory alebo videorozhovory.</li>
+                <li class="pb-2">Neopúšťajte svoj domov, pokiaľ to nie je úplne nevyhnutné.</li>
+            </ol>
+
+            <p class="text-xl">
+                Majte na pamäti, že neexistuje žiadne správne alebo nesprávne množstvo opatrení. Príjmajte iba tie opatrenia,
+                ktoré vám vyhovujú, a ktoré neohrozujú vaše živobytie. <strong>Neukončujte tým svoju prácu!</strong>
```

Comment:
> Neukončujte tým svoju prácu!  
  
This sounds more like "Don't end your work with this" rather than "Don't quit your job over this".